spec/fixtures/ial2_test_credential_poor_quality.yml

```diff
@@ -0,0 +1,20 @@
+portrait_match_results:
+  # returns the portrait match result
```

Comment:
please share the difference between this file and spec/fixtures/ial2_test_credential_no_liveness.yml ?

Reply:
Ah, thanks for catching that. I'm not sure how that happened, but corrected it [here](https://github.com/18F/identity-idp/pull/9975/commits/e1428dd0de40bc5f088fe27c707b4f6b9306d256).